### ISSUE TITLE
ABI Break introduced in 19.3.4 release tag.

### ISSUE THREAD

**Issue by johnbash** [intel/gmmlib#65]
GMMLib is creating revert for ABI break introduced in 19.3.4 release to save package maintenance cost to open source distributions. SO version name will be updated back to libigdgmm.so.11 in future release. Please discard libigdgmm.so.12, no support will continue in future for same.
 
All existing functionality in libigdgmm.so.11 will be maintained as is.

**Comment by jbeich** [comment]
Note, ABI revert broke API that NEO relied on. See https://github.com/intel/compute-runtime/pull/244 for a fix.
```c++
In file included from runtime/kernel/kernel.cpp:10:
core/gmm_helper/gmm_helper.h:42:129: error: use of undeclared identifier 'GmmDestroy'
    static std::unique_ptr<GmmClientContext> (*createGmmContextWrapperFunc)(HardwareInfo *, decltype(&InitializeGmm), decltype(&GmmDestroy));
                                                                                                                                ^
core/gmm_helper/gmm_helper.h:51:15: error: use of undeclared identifier 'GmmDestroy'
    decltype(&GmmDestroy) destroyGmmFunc;
              ^
```

**Comment by JacekDanecki** [comment]
We're preparing fix in Neo repo, it'll be merged soon.

**Comment by JacekDanecki** [comment]
With [commit](https://github.com/intel/compute-runtime/commit/87e948eef584c7666d4dd7ccf0b5384077deedbf) Neo switched to gmmlib 19.4.1.

**Comment by johnbash** [comment]
Fixed the issue in intel-gmmlib-19.4.1 release.
